src/shared/db.ts

```diff
@@ -642,4 +644,15 @@ export class DB {
 
         return entry
     }
+
+    public async fetchEventDefinitions(): Promise<EventDefinitionType[]> {
```

Comment:
I guess these seem to only be used for tests? 

Also a minor thing is that this has same tag as a different query (one that filters by team)

Reply:
yep, they're only for tests. There are a few other fetch* functions like that in the db class, like `fetchElements` or `fetchEvents` that return everything without filters. They're useless in prod, but quite nice in tests. They could possibly be refactored to another file, but... not in this PR :)